posts/2023-01-20-types-announcement.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "Officially announcing the types team"
+author: Jack Huey
+description: "An overview of the new types team"
+team: The Types Team <https://github.com/rust-lang/types-team>
+---
+
+Oh hey, it's [another](https://blog.rust-lang.org/inside-rust/2022/09/29/announcing-the-rust-style-team.html) new team announcement. But I will admit: if you follow the [RFCs repository](https://github.com/rust-lang/rfcs/pull/3254), the [Rust zulip](https://rust-lang.zulipchat.com/#narrow/stream/144729-t-types), or were particularly observant on the [GATs stabilization announcement post](https://blog.rust-lang.org/2022/10/28/gats-stabilization.html), then this *might* not be a surprise for you. In fact, this "new" team was officially established at the end of May last year.
+
+There are a few reasons why we're sharing this post now (as opposed to months before or...never). First, the team finished a three day in-person/hybrid meetup at the beginning of December and we'd like to share the purpose and outcomes of that meeting. Second, posting this announcement now is just around 7 months of activity and we'd love to share what we've accomplished within this time. Lastly, as we enter into the new year of 2023, it's a great time to share a bit of where we expect to head in this year and beyond.
+
+## Backgound - How did we get here?
```

Comment:
I am confused how this section relates to the rest of the post. Most of it looks like project management stuff, which, while important, doesn't seem under the mandate of T-types. Maybe strip this down to just talk about the unsoundness issues?

Reply:
Yeah, I originally was going for "these are the things we've done in the past and why it hasn't been enough", but you're right that it doesn't quite fit the feel.

I'll trim it down.

Reply:
I trimmed it down a bit. What do you think now?